src/store/dashboard/ocpCloudDashboard/ocpCloudDashboardWidgets.ts

```diff
@@ -15,13 +16,13 @@ const getId = () => currrentId++;
 
 export const costSummaryWidget: OcpCloudDashboardWidget = {
   id: getId(),
-  titleKey: 'ocp_cloud_dashboard.cost_title',
+  titleKey: messages.OCPDailyUsageAndRequestComparison,
```

Comment:
This doesn't look like the correct title.

The old key was ""All cloud filtered by OpenShift cost"

Reply:
Note that this widget is temporarily hidden from the overview. Folks are working on the backend APIs to support this view.

Reply:
Done